addons/modules/functions/fnc_moduleCrewToGunner.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to have AI crew take the place of an incapacitated gunner.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleCrewToGunner
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+private _vehicle = attachedTo _logic;
+deleteVehicle _logic;
+
+if (isNull _vehicle) exitWith {
+    [LSTRING(NoObjectSelected)] call EFUNC(common,showMessage);
+};
+
+if !(alive _vehicle) exitWith {
```

Comment:
Move the `!` inside the parenthses.

Reply:
https://github.com/zen-mod/ZEN/pull/423/commits/f588e0d79991a04accbb33264a3219e3d2b200c7